doc/funcref.sgml

```diff
@@ -2920,6 +2923,26 @@ used in presence of a prototype.
 </quote>
 
 
+<sect1>detect_c64dtv<label id="detect_c64dtv"><p>
+
+<quote>
+<descrip>
+<tag/Function/Check for the presence of the C64DTV.
+<tag/Header/<tt/<ref id="accelerator.h" name="accelerator.h">/
+<tag/Declaration/<tt/unsigned char detect_c64dtv (void);/
+<tag/Description/The function returns a 1 if a C64DTV has been detected.
+<tag/Notes/<itemize>
+<item>The function is specific to the C64.
+</itemize>
+<tag/Availability/cc65 (not all platforms)
+<tag/See also/
+<ref id="get_scpu_speed" name="get_c64dtv_speed">,
+<ref id="set_scpu_speed" name="set_c64dtv_speed">,
```

Comment:
The ID strings, in the references, are wrong.  (A copy-and-paste error from the SCPU functions.)